src/platform/packages/shared/kbn-ambient-ui-types/index.d.ts

```diff
@@ -25,6 +25,24 @@ declare module '*.svg' {
   export default content;
 }
 
+declare module '*.jpeg' {
+  const content: string;
+  // eslint-disable-next-line import/no-default-export
+  export default content;
+}
+
+declare module '*.jpg' {
+  const content: string;
+  // eslint-disable-next-line import/no-default-export
+  export default content;
+}
+
```

Comment:
Added these for parity with the webpack config, but let me know if you want me to remove these since they're not actively being used in this PR.